pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -346,102 +397,42 @@ func (p *provider) Create(machine *clusterv1alpha1.Machine, _ *cloudprovidertype
 		return nil, fmt.Errorf("dataVolumeName size %v, is bigger than 63 characters", len(dataVolumeName))
 	}
 
+	cpu := resourceRequirements.Requests[corev1.ResourceCPU]
 	virtualMachine := &kubevirtv1.VirtualMachine{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      machine.Name,
-			Namespace: c.Namespace,
+			Name:      c.VirtualMachineName,
+			Namespace: metav1.NamespaceDefault,
 			Labels: map[string]string{
-				"kubevirt.io/vm": machine.Name,
+				"kubevirt.io/vm": c.VirtualMachineName,
```

Comment:
We are missing the labels coming from the Preset here (from line 372).

Reply:
Preset labels are only required to be added in `VirtualMachineInstanceTemplateSpec` (line 414)

Reply:
perfect.

Reply:
I still think that it's a good idea to have the same labels in the VM and in the VMI.